packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -432,14 +434,41 @@ export function mapCustomChainToNetworkData(chain: ChainWithRpcUrl) {
   explorerUrls[chain.chainID] = chain.explorerUrl
 }
 
-function isL1Chain(chain: L1Network | L2Network): chain is L1Network {
+export function isL1Chain(chain: L1Network | L2Network): chain is L1Network {
   return !chain.isArbitrum
 }
 
 function isArbitrumChain(chain: L1Network | L2Network): chain is L2Network {
   return chain.isArbitrum
 }
 
+// given an Arbitrum chain id, it will get all the valid, whitelisted Orbit chain id's that are valid for teleportation
+// eg. Given "Arbitrum Sepolia" => [Stylus]
+export function getOrbitChainIdsFromArbitrumChainIdForTeleport(
```

Comment:
This seems unnecessarily complicated. Why not just keep a map of L1 chain to L3 chains and then return those L3 chains as destination chains for that L1?